msm8936.c

```diff
@@ -49,137 +49,139 @@
-	 { "gcc_blsp1_qup6_spi_apps_clk", &gcc, 161 },
-	 { "gcc_blsp1_qup6_i2c_apps_clk", &gcc, 162 },
-	 { "gcc_camss_ahb_clk", &gcc, 168 },
-	 { "gcc_camss_top_ahb_clk", &gcc, 169 },
-	 { "gcc_camss_micro_ahb_clk", &gcc, 170 },
-	 { "gcc_camss_gp0_clk", &gcc, 171 },
-	 { "gcc_camss_gp1_clk", &gcc, 172 },
-	 { "gcc_camss_mclk0_clk", &gcc, 173 },
-	 { "gcc_camss_mclk1_clk", &gcc, 174 },
-	 { "gcc_camss_mclk2_clk", &gcc, 445 },
-	 { "gcc_camss_cci_clk", &gcc, 175 },
-	 { "gcc_camss_cci_ahb_clk", &gcc, 176 },
-	 { "gcc_camss_csi0phytimer_clk", &gcc, 177 },
-	 { "gcc_camss_csi1phytimer_clk", &gcc, 178 },
-	 { "gcc_camss_jpeg0_clk", &gcc, 179 },
-	 { "gcc_camss_jpeg_ahb_clk", &gcc, 180 },
-	 { "gcc_camss_jpeg_axi_clk", &gcc, 181 },
-	 { "gcc_camss_vfe0_clk", &gcc, 184 },
-	 { "gcc_camss_cpp_clk", &gcc, 185 },
-	 { "gcc_camss_cpp_ahb_clk", &gcc, 186 },
-	 { "gcc_camss_vfe_ahb_clk", &gcc, 187 },
-	 { "gcc_camss_vfe_axi_clk", &gcc, 188 },
-	 { "gcc_camss_csi_vfe0_clk", &gcc, 191 },
-	 { "gcc_camss_csi0_clk", &gcc, 192 },
-	 { "gcc_camss_csi0_ahb_clk", &gcc, 193 },
-	 { "gcc_camss_csi0phy_clk", &gcc, 194 },
-	 { "gcc_camss_csi0rdi_clk", &gcc, 195 },
-	 { "gcc_camss_csi0pix_clk", &gcc, 196 },
-	 { "gcc_camss_csi1_clk", &gcc, 197 },
-	 { "gcc_camss_csi1_ahb_clk", &gcc, 198 },
-	 { "gcc_camss_csi1phy_clk", &gcc, 199 },
-	 { "gcc_camss_csi2_clk", &gcc, 227 },
-	 { "gcc_camss_csi2_ahb_clk", &gcc, 228 },
-	 { "gcc_camss_csi2phy_clk", &gcc, 229 },
-	 { "gcc_camss_csi2rdi_clk", &gcc, 230 },
-	 { "gcc_camss_csi2pix_clk", &gcc, 231 },
-	 { "gcc_pdm_ahb_clk", &gcc, 208 },
-	 { "gcc_pdm2_clk", &gcc, 210 },
-	 { "gcc_prng_ahb_clk", &gcc, 216 },
-	 { "gcc_camss_csi1rdi_clk", &gcc, 224 },
-	 { "gcc_camss_csi1pix_clk", &gcc, 225 },
-	 { "gcc_camss_ispif_ahb_clk", &gcc, 226 },
-	 { "gcc_boot_rom_ahb_clk", &gcc, 248 },
-	 { "gcc_crypto_clk", &gcc, 312 },
-	 { "gcc_crypto_axi_clk", &gcc, 313 },
-	 { "gcc_crypto_ahb_clk", &gcc, 314 },
-	 { "gcc_oxili_timer_clk", &gcc, 489 },
-	 { "gcc_oxili_gfx3d_clk", &gcc, 490 },
-	 { "gcc_oxili_ahb_clk", &gcc, 491 },
-	 { "gcc_oxili_gmem_clk", &gcc, 496 },
-	 { "gcc_venus0_vcodec0_clk", &gcc, 497 },
-	 { "gcc_venus0_core0_vcodec0_clk", &gcc, 440 },
-	 { "gcc_venus0_core1_vcodec0_clk", &gcc, 441 },
-	 { "gcc_venus0_axi_clk", &gcc, 498 },
-	 { "gcc_venus0_ahb_clk", &gcc, 499 },
-	 { "gcc_mdss_ahb_clk", &gcc, 502 },
-	 { "gcc_mdss_axi_clk", &gcc, 503 },
-	 { "gcc_mdss_pclk0_clk", &gcc, 504 },
-	 { "gcc_mdss_pclk1_clk", &gcc, 442 },
-	 { "gcc_mdss_mdp_clk", &gcc, 505 },
-	 { "gcc_mdss_vsync_clk", &gcc, 507 },
-	 { "gcc_mdss_byte0_clk", &gcc, 508 },
-	 { "gcc_mdss_byte1_clk", &gcc, 443 },
-	 { "gcc_mdss_esc0_clk", &gcc, 509 },
-	 { "gcc_mdss_esc1_clk", &gcc, 444 },
-	 { "gcc_bimc_clk", &gcc, 340 },
-	 { "gcc_bimc_gpu_clk", &gcc, 343 },
-	 { "gcc_bimc_ddr_ch0_clk", &gcc, 346 },
-	 { "gcc_cpp_tbu_clk", &gcc, 233 },
-	 { "gcc_mdp_rt_tbu_clk", &gcc, 238 },
-	 { "wcnss_m_clk", &gcc, 408 },
+	 { "gcc.mux_gp1_clk", &gcc.mux, 16 },
```

Comment:
Did you intend to `s/gcc_/gcc./` in the string? I don't see this in other platforms.

Reply:
Stupid me and stupid regexps. I'll fix that.